### PR TITLE
Update from facebook 1ee4edd286a3

### PR DESCRIPTION
**Pull request by bwasti**

### CODE REVIEW

**Review by ezyang — APPROVED** [review]

**Comment by SsnL** [comment]
Can we fix the lint please?

File: torch/jit/__init__.py [hunk 1]
```diff
@@ -717,7 +717,8 @@ class _ConstSequential(_ConstModuleList):
     def __init__(self, mods):
         super(_ConstSequential, self).__init__(mods._modules.values())
 
-    @script_method
+    # commenting script_method until T29956663 is resolved
+    #@script_method
```
Comment (apaszke) [inline torch/jit/__init__.py]: Why did this change? Can you open a public issue describing the problem?

Comment (soumith) [reply torch/jit/__init__.py]: this change can be removed. I'll push a patch shortly to address this. We need to wrap ConstSequential as a string source, to get around opt mode only shipping `.pyc` files (and no .py files).

This change as-is will also fail oss contbuilds

**Comment by bwasti** [comment]
@SsnL the goal of this diff is purely to sync codebases as minimally as possible

**Comment by SsnL** [comment]
@bwasti I understand, but it is just a single small lint, probably fixed by 1 character (I forgot the exact error)... It is fine now after you reverted the offending change. But next time, maybe we can open a sister PR that fixes only lints, and merge that as soon as the sync PR lands, so that other PRs' test won't be influenced?

**Review by soumith — APPROVED** [review]

**Pull request merged**